tests/e2e/cypress/integration/spanish.spec.js

```diff
@@ -0,0 +1,48 @@
+describe("Spanish language", () => {
+  beforeEach(() => {
+    cy.visit("/")
+  })
+
+  it("Gives user Spanish language option", () => {
+    cy.get("#header").should("contain", "Español")
+    cy.get("#header").should("not.contain", "English")
+  })
+
+  it("Changes the language to Spanish and does not change URL", () => {
+    cy.location("pathname").should("eq", "/")
+
+    cy.contains("Español")
+      .click()
+
+    cy.get("#header").should("not.contain", "Español")
+    cy.get("#header").should("contain", "English")
+    cy.location("pathname").should("eq", "/")
+  })
+
+  it("Changes the language from Spanish back to English", () => {
+    cy.contains("Español")
+      .click()
+
+    cy.get("#header").should("not.contain", "Español")
+    cy.get("#header").should("contain", "English")
+    cy.contains("English")
+      .click()
+
+    cy.get("#header").should("contain", "Español")
+    cy.get("#header").should("not.contain", "English")
+  })
+
+  it("Changes the language to Spanish from any page", () => {
+    cy.get(".buttons .btn")
+      .first()
+      .click()
+      .location("pathname").should("eq", "/abc")
```

Comment:
Should the `/abc` part be driven by the `transit-agencies.json` fixture? I think we could use the `slug` property.